Philip_Cron/assignments/apple_tree/apple_tree_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'apple_tree'
+RSpec.describe Apple_Tree do
+    before do
+        @tree1 = Apple_Tree.new
+    end
+    it 'should have a height and age' do
+        expect(@tree1.height).to eq(150)
+        expect(@tree1.age).to eq(9)
+    end
+    it 'should have a count of how many apples is on it' do
+        expect(@tree1.apple_count).to eq(1)
+    end
+    it 'should have a method called year_gone_by, which ages the tree by 1 year affecting its height' do
+        @tree1.year_gone_by
+        expect(@tree1.age).to eq(@tree1.age)
+        expect(@tree1.height).to eq(@tree1.height)
```

Comment:
unless the tree doesn't have a getter these tests could never fail.  You should write the actual number you expect in the .to eq.  So if you start at height 150, age 9, after year gone by they should equal 151 and 10

expect(@tree1.age).to eq(10)